src/types.ts

```diff
@@ -25,3 +27,15 @@ export interface ImageRenderOptions extends RenderOptions {
   fullPageImage?: boolean;
   scaleImage?: number;
 }
+
+export type SanitizeRequest = {
+  filename: string;
+  content: string;
```

Comment:
will we ever want this to support images/binary?

Reply:
base64 ftw! :) I figured that a multipart request with `file` and `config` sections is the proper way of implementing this. back to codingg

Reply:
refactored to a multipart request